exporter.go

```diff
@@ -117,5 +182,6 @@ func main() {
 			</body>
 			</html>`))
 	})
+	log.Printf("NGINX Prometheus Exporter has successfully started. Metrics available at http://localhost%v%v", *listenAddr, *metricsPath)
```

Comment:
Not sure if adding `localhost` here might generate a confusion when this is not deployed locally and the users will try to access it via localhost. I would say is just easier to write `"Metrics available at :%v%v, *listenAddr, *metricsPath"` Does it make sense? 🤔